Automattic/wp-calypso
# Subscribers Page: Add strings for translation

⚠️ Please do not merge this PR. ⚠️ 

Resolves https://github.com/Automattic/wp-calypso/issues/77676.

TODO:

- [x] add screenshots to this PR before merging

| Subscribers page | Subscription details | "Grow your subscribers" page |
|--------|--------|--------|
| ![Markup on 2023-06-06 at 11:08:31](https://github.com/Automattic/wp-calypso/assets/25105483/d5b194b8-1698-480c-b37b-03bbda339da5) | ![Markup on 2023-06-06 at 11:10:05](https://github.com/Automattic/wp-calypso/assets/25105483/122388c2-e994-4504-a5fb-93ab343d6e63) | ![Markup on 2023-06-06 at 11:12:09](https://github.com/Automattic/wp-calypso/assets/25105483/bac419b5-43da-4a0f-ab4a-f76dd599b05c) | 

## Proposed Changes

The PR introduces strings that are related to the:

* Subscribers page
* "Grow your subscribers" section and page
* "Subscription details" page (even though this is not part of the Milestone I., I decided to include the strings in case there will be some changes needed related to handling single / multiple paid subscriptions per user)

The PR includes also strings like:

* Strings for "confirmation dialogs" - messages that the site owner will see when they click on the "Remove" button. 
* "Manage" button label (not in the design, but we may need it)
* "Open rate" string (and similar) that aren't planned to be delivered in Milestone I

ℹ️ The PR does not include strings related to the subscribers import screens.

## Testing Instructions

1. Review the strings, check for typos and compare them with the strings in the design (h8tMpJlCqNFemEerU9owHI-fi-1737_56094).
2. Make sure the PR doesn't introduce any regressions.

## Pre-merge Checklist
